### ISSUE TITLE
New buzzwords Deep Learning Dissected: The Awesome Potential of Artificial Intelligence

### ISSUE THREAD

**Issue by postmillenial** [postmillenial/reading-list#627]
Deep Learning Dissected: The Awesome Potential of Artificial Intelligence<br>
http://ift.tt/2y63qYx<br><img width="640" height="426" src="http://ift.tt/2gp04s6" class="webfeedsFeaturedVisual wp-post-image wp-stateless-item" style="display: block; margin: auto; margin-bottom: 20px;" data-image-size="large" data-stateless-media-bucket="cdn.thenewstack.io" data-stateless-media-name="media/2017/10/a0fe7568-child-659283_640.jpg"><p><em>This post is the first of a nine-part series, called &ldquo;Deep Learning Dissected&rdquo; contributed by IBM, that will explore the challenges in adopting deep learning-based cognitive systems.&nbsp;</em></p>
<a href="http://www.ibm.com" class="clearfix infoBlock" target="_blank"></a>
<div class="infoBlockImageBlock"><img class="infoBlockImage" src="http://ift.tt/2yAMmx2" style="border-radius: 50%;"></div>
<div class="infoBlockTextBlock">
<div class="infoBlockTitle">Adel El-Hallak</div>
<div class="infoBlockText">Adel El-Hallak is the Director of Product Management for Deep Learning within IBM&rsquo;s Cognitive Systems unit. Adel&rsquo;s mission is to democratize and operationalize deep learning so it can be more easily consumed in order to augment human intelligence. Adel&rsquo;s previous roles include leadership positions across sales and marketing for high-performance computing and advanced analytics. Adel holds a bachelor&rsquo;s degree in computer science from McGill University in Canada and an M.B.A. from Warwick Business School in the U.K.</div>
</div>
<p>My daughter recently asked me, &ldquo;Daddy, what do you do?&rdquo; Needing to distill my job for a four-year-old to understand, I responded with &ldquo;I help computers see.&rdquo;</p>
<p>&ldquo;But they don&rsquo;t have eyes,&rdquo; she replied.</p>
<p>&ldquo;No, but they can process convolutional neural network and classify images,&rdquo; I replied.</p>
<p>With a toddler&rsquo;s typical pragmatism she asked: &ldquo;Can it see me?&rdquo;</p>
<p>Never wanting to disappoint her, I responded: &ldquo;of course!&rdquo;</p>
<p>As she impatiently stared at my computer screen, I downloaded hundreds of pics of our two daughters from my iPhone, categorized them, trained an image classification neural network using <a href="http://ift.tt/1gG1byZ" class="ext-link" rel="external nofollow">Caffe</a>, and exposed the model as a REST API. We spent the rest of the afternoon scoring the model to see if it correctly identified our two daughters.</p>
<p><a href="http://ift.tt/2yAMn44" class="ext-image" rel="external nofollow"><img class="aligncenter size-full wp-image-3214663" src="http://ift.tt/2yAMn44" width="2551" height="1237" data-id="3214663"></a></p>
<p>Toddlers aren&rsquo;t the only ones excited about the potential of AI, but realizing business value with deep learning comes with its challenges. Scarce market skills for data scientists, a cumbersome data preparation process, a growing number of open source models and frameworks spurring a vibrant, yet nascent, ecosystem bar broader adoption. But those that can overcome these barriers are disrupting industries, reaping revenue streams through new business models, and improving client experiences.</p>
<p><a href="http://ift.tt/2yAHckJ" class="ext-image" rel="external nofollow"><img class="aligncenter size-full wp-image-3214664" src="http://ift.tt/2yAHckJ" width="1500" height="1269" data-id="3214664"></a></p>
<p>I find myself in a privileged position to confront this challenge. My first exposure to advanced analytics came early in my career. As a young technical seller, I configured GPU-enabled Linux clusters to help academia advance scientific research, chemical petroleum companies simulate and visualize oil reservoirs and countries to make more accurate weather predictions. GPUs are now being leveraged to process massive amounts of data, and this early career experience gave me hands-on foundational knowledge of accelerators and scale out clusters that have become integral to today&rsquo;s deep learning solutions.</p>
<p>I then spent time in growth markets, such as Africa, where I witnessed developing nations go from building basic infrastructure to adopting emerging technologies such as mobile and cloud giving newfound access to mobile payment systems, changing lives of millions. As a part of this process, I helped overcome challenges that impeded adoption of technology, and after living &ldquo;in the trenches,&rdquo; I understood what some of the blockers to the process can be.</p>
<p>Most recently, as a global product director, I spent time a lot of time with data scientists, developers and business leaders, fostering discussion with them on what is enabling and inhibiting them from realizing value through machine learning.</p>
<p>These series of posts will seek to discuss some of my findings; what I&rsquo;m seeing and hearing are the most challenging inhibitors to deep learning adoption within organizations of all sizes. I&rsquo;ll share insights I&rsquo;ve learned from experts around the globe on such important topics as:</p>
<ul>
<li>Understanding what use cases can be derived from your organization&rsquo;s data, and driving a data-driven culture with leadership buy-in to enable a multidisciplinary team.</li>
<li>Building the right infrastructure that supports this cyclical process in a rapidly evolving technology space.</li>
<li>Facilitating the process of transforming data for deep learning, and making sure data can be labeled in a time-effective manner, a stage that typically takes 80 percent of a data scientist&rsquo;s time. Establishing the right models and frameworks to pick, optimizing the models, and reducing training times. Enabling the consumption of trained algorithms across different deployment models and applying deep learning outputs to drive organizational success.</li>
<li>Maintaining model accuracy and fine-tuning it with additional inputs.</li>
</ul><p><a href="http://ift.tt/2yAHesR" class="ext-image" rel="external nofollow"><img class="aligncenter size-full wp-image-3214665" src="http://ift.tt/2yAHesR" width="1280" height="720" data-id="3214665"></a></p>
<p>All of these important processes need to be carefully considered before deep learning can successfully be&nbsp;applied within an organization, whether it be the largest enterprise or a small to mid-sized business.</p>
<p><a href="http://ift.tt/2yAHfwV" class="ext-image" rel="external nofollow"><img class="aligncenter size-full wp-image-3214666" src="http://ift.tt/2yAHfwV" width="287" height="210" data-id="3214666"></a></p>
<p>Let me know on Twitter at <a href="https://twitter.com/adelhallak" class="ext-link" rel="external nofollow">@adelhallak</a> or <a href="http://ift.tt/2gpji0T" class="ext-link" rel="external nofollow">LinkedIn.</a></p>
<p class="attribution">Feature image <a href="http://ift.tt/1TVuldu" class="ext-link" rel="external nofollow">via</a> Pixabay.</p>
<p>The post <a rel="nofollow" href="http://ift.tt/2y63qYx">Deep Learning Dissected: The Awesome Potential of Artificial Intelligence</a> appeared first on <a rel="nofollow" href="https://thenewstack.io">The New Stack</a>.</p>
<br><br>
via Blog &ndash; The New Stack https://thenewstack.io<br>
October 09, 2017 at 12:00PM
